src/Data/Managers/V2/Miscellaneous/ControlAgent/useControlAgent.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  UseMutationOptions,
+  UseMutationResult,
+  useMutation,
+  useQueryClient,
+} from "@tanstack/react-query";
+import { usePost } from "../../helpers";
+
+/**
+ * Interface for control agents parameters
```

Comment:
We might use the oportunity to rename this. 
`ControlAgent` can mean about controling anything for the Agent, while this hook specifically Pauses/Unpauses the agent. `usePauseAgent` sounds better to me, any opinion on this?